docs/issuing_certificates.md

```diff
@@ -0,0 +1,19 @@
+# Issuing certificates
+
+## Securing OpenShift `Route`s with ACME (Let's Encrypt)
+
+We manage and deploy cluster wide deployment of [OpenShift ACME](https://github.com/tnozicka/openshift-acme) controller. This operator facilitates you Let's Encrypt certificate provisioning in any namespace on the cluster. Any `Route` resource is eligible, please use the following annotation to mark the route as managed by this operator:
```

Comment:
> This operator facilitates you Let's Encrypt certificate provisioning  
change to: 
> This operator facilitates Let's Encrypt certificate provisioning  

And maybe add a link for let's encrypt.

Reply:
Updated